### ISSUE TITLE
[7.10.1]빌드시 오류가 발생합니다.

### ISSUE THREAD

**Issue by bean5389** [naver/nam-sdk-ios#25]
안녕하세요. 빌드과정에서 오류가 발생해 문의드립니다.

환경: Xcode 15.4

```
pod 'NAMSDK'
pod 'NAMSDK/MediationNDA'

```

```
import GFPSDK
.....
func configure(placementId: String) {
    
    GFPAdManager.setup(withPublisherCd: "####", target:self) { (error : GFPError?) in
      print("Setup Eror: \(String(describing: error?.description))")
    }
    
  }
```

아래와 같이 빌드 시 오류가 발생합니다.
![image](https://github.com/user-attachments/assets/a406da91-f5fb-4d29-b6a7-d9ecf69e0fb7)

**Comment by shhan-sys** [comment]
안녕하세요, @bean5389 님
15.4 버전으로 빌드 테스트해보니 
```
Installing NAMSDK (7.10.2)
Installing Naver-Ads-Services (1.0.8)
```
해당 버전으로 설치 뒤 올바르게 잘 빌드가 되는 것으로 보입니다.
옵션 및 빌드 세팅 차이로 인해서 빌드가 안되는 것으로 보이는데요

```
rm -rf Pods Podfile.lock 
pod install repo-update

```
재설치 이후에도 빌드가 되지 않으신다면 옵션 확인을 위해 프로젝트 파일 공유 등이 필요해보입니다

**Comment by shhan-sys** [comment]
@bean5389 님, 공유된 팟 파일 참고시 use_frameworks! 를 사용하지 않는 형태로 보이는데요,
혹시 해당 옵션을 사용하여도 빌드가 되지 않을까요?

**Comment by bean5389** [comment]
기존에 사용중인 라이브러리들로인해 해당 옵션 추가시 다른 오류가 발생합니다.

**Comment by shhan-sys** [comment]
네, 확인 감사합니다.
동일하게 use_frameworks! 옵션 없이 새 프로젝트 빌드 시에는 빌드가 잘 되는 듯 싶은데요,
`
source 'https://github.com/CocoaPods/Specs.git'


# Uncomment the next line to define a global platform for your project
platform :ios, '15.0'

target 'test' do
  # Comment the next line if you don't want to use dynamic frameworks
  #use_frameworks!

  # Pods for test
  pod 'NAMSDK'
  pod 'NAMSDK/MediationNDA'


end

inhibit_all_warnings!
post_install do |installer|
    installer.pods_project.targets.each do |target|
        target.build_configurations.each do |config|
            config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '13.0'
            config.build_settings["EXCLUDED_ARCHS[sdk=iphonesimulator*]"] = "arm64"
            config.build_settings['CODE_SIGNING_ALLOWED'] = 'NO'
        end
    end
end
`
혹시 새 프로젝트로 빌드 테스트 시에도 동일한 현상이 발생할까요?

**Comment by bean5389** [comment]
새 프로젝트로 빌드시에는 발생하지 않습니다.

**Comment by shhan-sys** [comment]
네, 마지막으로 타겟의 other flags에 -ObjC ,  Build Setting에서 Defines Module 여부 확인 한 번 부탁드리겠습니다

**Comment by bean5389** [comment]
Other Linker Flags 에 -Objc 추가되어있고
Defines Module YES로 설정되어있습니다.

**Comment by shhan-sys** [comment]
음..그럼 프로젝트 참조 설정에 문제가 있는 듯 한데요
제가 프로젝트를 직접 열어볼 수 있는 환경이 아니라 이슈 진단하기가 쉽지 않은 듯 합니다. 
차주에 이슈있는 Enum 통합이 가능할 지 논의해보고 노티 드리겠습니다

**Comment by bean5389** [comment]
네 확인 감사합니다.
이후에는 광고사 통해서 문의드리겠습니다.
